app/models/miq_expression.rb

```diff
@@ -1557,6 +1557,30 @@ def self._quick_search?(e)
     end
   end
 
+  def custom_attribute_columns
+    exp ? self.class._custom_attribute_columns(exp) : []
+  end
+
+  def self._custom_attribute_columns(expression)
+    result = []
```

Comment:
nit: (you can probably ignore these comments)

Can you just define this without `result` variable?